training/distributed_training/pytorch/model_parallel_v2/shared-scripts/arguments.py

```diff
@@ -27,6 +27,9 @@ def parse_args():  # pylint: disable=too-many-statements
     opt_grp.add_argument("--seed", type=int, default=12345)
     opt_grp.add_argument("--same_seed", type=int, default=0)
     opt_grp.add_argument("--bf16", default=1, type=int, help="automatic mixed precision training")
+    opt_grp.add_argument("--fp8", default=1, type=int, help="fp8 mixed precision training")
+    opt_grp.add_argument("--fp8_amax_history_len", default=1024, type=int, help="amax history length")
```

Comment:
Do we have a particular reason to use this default value? Seems some other less values, 1 or 32, are also used by others.

Reply:
`fp8_amax_history_len=1024` is the default set by Transformer Engine, but we could change to another value if we want.
@cavdard